### PR TITLE
pep8 addusers.py

### PR DESCRIPTION
**Pull request by duaatashk**
fixed all the errors with addusers.py with the PR

### CODE REVIEW

**Comment by kamfonik** [comment]
This should pass the checks once #26 is merged, so we won't worry about that for now.

File: addusers.py [hunk 1]
```diff
@@ -130,10 +134,9 @@ def create_project(self, project, quotas):
 
     def create_user(self, user, project):
         """Create a new OpenStack user from an addusers.User
-
```
Comment (duaatashk) [inline addusers.py]: i fixed this. 

you're right i can have the blank line. i don't remember why i deleted it.. :/

**Comment by kamfonik** [comment]
I resolved merge conflicts since they were caused by some fixes I made earlier.

**Comment by xuhang57** [comment]
CI fails.

**Comment by kamfonik** [comment]
@xuhang57  it should pass once #32 is merged - we removed the ignored files from setup.cfg in this PR, so it is picking up the pep8 issues fixed by #32.  I will add a space in the README or something to trigger a new build after #32 is merged.

**Comment by kamfonik** [comment]
Not quite, it looks like.  @duaatashk the CI checks are still picking up 3 PEP8 errors in addusers.py, can you fix them:

```
./addusers.py:357:5: E303 too many blank lines (2)
./addusers.py:359:59: W291 trailing whitespace
./addusers.py:380:55: W291 trailing whitespace
```

**Comment by duaatashk** [comment]
@kamfonik That's weird. I checked it again and it didn't give me any errors. I checked the lines in your comment above and I didn't have any whitespace or blank lines. I'll push it again. I'm guessing maybe I pushed the wrong thing?

**Comment by kamfonik** [comment]
@duaatashk did you pull the additional commits? (You will need to do that before it lets you push any more commits to this branch)

It may have been "unfixed" during the merge conflict resolution.

**Comment by duaatashk** [comment]
pulling... i always forget to pull... fixed the errors. should be good now

**Pull request merged**